packages/headless/src/app/recommendation-engine/recommendation-engine.ts

```diff
@@ -87,6 +89,16 @@ export function buildRecommendationEngine(
 
   const engine = buildEngine(augmentedOptions, thunkArguments);
 
+  const {pipeline, searchHub} = options.configuration;
+
+  if (pipeline) {
```

Comment:
nit: could use `isNullOrUndefined` from bueno instead.

If somebody decides to call their search hub `0`.... 🤷